app/lib/gen/strings_en.g.dart

```diff
@@ -641,6 +641,7 @@ class TranslationsSettingsTabSendEn {
   // Translations
   String get title => 'Send';
   String get shareViaLinkAutoAccept => 'Automatically accept requests in "Share via link" mode';
+  String get privacyProtectionMode => 'Privacy Protection Mode: No Photo Library Access';
```

Comment:
Don't edit the generated file, edit the JSON file first, then run `dart run slang`

Reply:
fixed